modules/backlogs/lib/open_project/backlogs/list.rb

```diff
@@ -81,9 +81,13 @@ def move_after(prev_id)
 
     protected
 
-    def assume_bottom_position
```

Comment:
The method `assume_bottom_position` was renamed or deleted (not sure what the intent is here) but it is still used in `#fix_own_work_package_position`: 
https://github.com/opf/openproject/blob/8868186415ae22b613f8f9b978c5111f8d1e43a1/modules/backlogs/lib/open_project/backlogs/list.rb#L145

And I do not see `set_list_position` being called either. Am I missing something?

Reply:
`Backlogs::List` [includes `acts_as_list`](https://github.com/opf/openproject/blob/dev/modules/backlogs/lib/open_project/backlogs/list.rb#L33).

Before the change `assume_bottom_position` of `acts_as_list` was overwritten. That original implementation, active again now, in turn [calls `set_list_position`](https://github.com/brendon/acts_as_list/blob/905dd30aacfcb245045643f1da577ea17e1ace27/lib/acts_as_list/active_record/acts/list.rb#L292-L294). So with the change, the saving of acts_as_list will now always use `update_column` instead of only doing it for `assume_bottom_position`.

Reply:
I just made the comment more explicit b3839d420b.